Jenkinsfile

```diff
@@ -22,7 +22,9 @@ pipeline {
                         ./pull.sh
                         cd ..
                         rm -rf .git .github .gitignore Jenkinsfile
-                        zip $OUTPUT_NAME.zip -9 -r ./
+                        cd ..
+                        mv $REPO_NAME $OUTPUT_NAME
+                        zip $OUTPUT_NAME.zip -9 -r $OUTPUT_NAME
```

Comment:
The output file (**$OUTPUT_NAME.zip**) should be moved to **$OUTPUT_NAME** directory, so that deploy  stage will find the zip file.  i.e. mv $OUTPUT_NAME.zip  $OUTPUT_NAME after line 27

Reply:
I restored the REPO directory and changed back into it, but left the zip 1 level higher.  Think this should also work?